### PR TITLE
Display list of pizza shop reviews by user

### PR DESCRIPTION
**Pull request by jchaing**
# Description

Displays list of Pizza Shop Reviews by the user in their dashboard

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by jchaing** [comment]
[Display personal reviews from individual pizza eater to dashboard](https://trello.com/c/qBajqwX6/30-display-personal-reviews-from-individual-pizza-eater-to-dashboard)

File: plza-fe/src/components/users/reviews/UserReview.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  const user = useSelector((state) => state.user);
+  const [toggleEdit, setToggleEdit] = useState(false);
+  const dispatch = useDispatch();
+  const [reviewToEdit, setReviewToEdit] = useState({});
+
+  const editReview = (review) => {
+    setReviewToEdit(review);
+  };
+
+  console.log(user);
```
Comment (agyin3) [inline plza-fe/src/components/users/reviews/UserReview.js:17]: Would it be possible to get this console.log removed?

**Comment by agyin3** [comment]
Really like the option of being able to edit the review directly from the user dashboard

File: plza-fe/src/components/users/reviews/UserReview.js (hunk 1 above)
Comment (jchaing) [reply plza-fe/src/components/users/reviews/UserReview.js]: Done. https://github.com/Lambda-School-Labs/lets-get-pizza-fe/pull/62/commits/1ecc211a05654ae0e2914f87ca6183a3060b2ad9

**Pull request merged**